code/controllers/subsystem/server_maint.dm

```diff
@@ -1,23 +1,39 @@
+#define PING_BUFFER_TIME 25
+
 SUBSYSTEM_DEF(server_maint)
 	name = "Server Tasks"
-	wait = 6000
-	flags = SS_NO_TICK_CHECK
+	wait = 6
+	flags = SS_POST_FIRE_TIMING|SS_FIRE_IN_LOBBY
+	priority = 10
+	var/list/currentrun
 
-/datum/controller/subsystem/server_maint/Initialize(timeofday)
```

Comment:
All arguments available to a proc should be present in all overrides, even if not used, THAT is enough of a reason to revert this change.